feature-detects/css/subpixelfont.js

```diff
@@ -20,8 +20,8 @@ define(['Modernizr', 'testStyles'], function(Modernizr, testStyles) {
     function(elem) {
       var subpixel = elem.firstChild;
       subpixel.innerHTML = 'This is a text written in Arial';
-      Modernizr.addTest('subpixelfont', window.getComputedStyle ?
-        window.getComputedStyle(subpixel, null).getPropertyValue('width') !== '44px'
+      Modernizr.addTest('subpixelfont', computedStyle ?
```

Comment:
same here

Reply:
three times is the removal charme I assume...